justfile

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2024 Phoenix R&D GmbH <hello@phnx.im>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+export DATABASE_URL := "postgres://postgres:password@localhost:5432/phnx_db"
+
+# run postgres via docker compose and apply migrations
+init-db: generate-db-certs
+    docker compose up --wait
+    cd backend && sqlx database create
```

Comment:
JFYI: every step is by default executed in its own shell, so `pwd` is not preserved.